tests/integration/targets/dns_config_info/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+- environment:
+    SC_HOST: "{{ sc_host }}"
+    SC_USERNAME: "{{ sc_username }}"
+    SC_PASSWORD: "{{ sc_password }}"
+
+  block:
+    - name: Retrieve info about DNS Config
```

Comment:
We can use api module to push server into known state, then we can check if exactly correct values are returned. 

What I hate is we need to restore server to original state - but is a test system for CI tests, so this restore is a minor thing.